extensions/attention-stream/scripts/generate-manifest.ts

```diff
@@ -27,6 +27,7 @@ async function main(manifestVersion, browser) {
   switch (manifestVersion) {
     case 2: {
       manifest["browser_action"] = {
+        default_icon: "images/RallyFlag-green.png",
```

Comment:
I think it's probably safe to do it like this, but since we're actually trying to differentiate based on browser and not manifest, seems better to set the `iconPath` separately based on browser, and use that variable here.

Reply:
Yes I feel like keeping the manifest vs. browser distinction clear will be important later, when Firefox moves to mv3.